extensions/amp-analytics/0.1/vendors/memo.js

```diff
@@ -0,0 +1,63 @@
+/**
+ * Copyright 2019 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {jsonLiteral} from '../../../../src/json';
+
+const MEMO_CONFIG = jsonLiteral({
+  'vars': {
+    'pid': '',
+    'published_at': '',
+    'authors': [],
+  },
+  'requests': {
+    'host': 'https://elsa.memoinsights.com',
+    'baseParams':
+      'amp=true' +
+      '&pid=${pid}' +
+      '&title=${title}' +
+      '&url=${canonicalUrl}' +
+      '&date=${published_at}' +
+      '&authors=${authors}' +
+      '&referrer=${documentReferrer}' +
+      '&page_url=${sourceUrl}' +
+      '&user_id=${clientId(memo_amp_id)}' +
+      '&domain=${canonicalHost}' +
+      '&v=amp',
+    'visible': '${host}/n?${baseParams}',
+    'timer': '${host}/t?${baseParams}&t=5000&e=5000',
+  },
+  'triggers': {
+    'visible': {
+      'on': 'visible',
+      'request': 'visible',
+    },
+    'timer': {
+      'on': 'timer',
+      'timerSpec': {
+        'interval': 5,
+        'maxTimerLength': 1200,
+      },
+      'request': 'timer',
+    },
+  },
+  'transport': {
+    'beacon': false,
+    'xhrpost': false,
+    'image': true,
+  },
```

Comment:
To echo my last comment: just to confirm that the server only handles get request. AMP handles the fallback to xhrpost and image if the browser doesn't support one transportation method. So we'd recommend using the default transport method if that's supported by server. Thanks.